### ISSUE TITLE
expo-image-picker >= 14.0.2 caused error TypeError: config_plugins_1.AndroidConfig.Permissions.withBlockedPermissions is not a function

### ISSUE THREAD

**Issue by coroo** [expo/expo#21573]
### Summary

I'm running with both this package version:
```
{
   "expo": "^47.0.0",
   "expo-image-picker": "~14.0.2",
}
```

And using expo to build the apk, then get error:
<a href="https://ibb.co/SV64HVj"><img src="https://i.ibb.co/6Dy7GDM/Screenshot-2023-03-07-at-10-46-33.png" alt="Screenshot-2023-03-07-at-10-46-33" border="0"></a>

### What platform(s) does this occur on?

Android, iOS

### SDK Version

47

### Environment

```
  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 13.1
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 14.17.6 - ~/.nvm/versions/node/v14.17.6/bin/node
      Yarn: 1.22.19 - ~/.nvm/versions/node/v14.17.6/bin/yarn
      npm: 6.14.15 - ~/.nvm/versions/node/v14.17.6/bin/npm
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.0.1, iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0
    IDEs:
      Android Studio: 2021.2 AI-212.5712.43.2112.8609683
      Xcode: 13.1/13A1030d - /usr/bin/xcodebuild
    npmPackages:
      @expo/metro-config: ^0.1.62 => 0.1.84 
      expo: ~47.0.12 => 47.0.13 
      react: 18.1.0 => 18.1.0 
      react-dom: 18.1.0 => 18.1.0 
      react-native: 0.70.5 => 0.70.5 
      react-native-web: ~0.18.7 => 0.18.12 
    npmGlobalPackages:
      eas-cli: 3.3.2
      expo-cli: 6.3.2
    Expo Workflow: managed
```

### Minimal reproducible example

1. Use the recommendation expo-image-picker version by SDK 47
```
{
   "expo": "^47.0.0",
   "expo-image-picker": "~14.0.2",
}
```
2. Build using eas (Error attached above)
4. Re-produce using local pre-build `npx expo prebuild --clean`, the error occurs also in local
```
- Config syncing
✖ Config sync failed
[stderr] 
TypeError: config_plugins_1.AndroidConfig.Permissions.withBlockedPermissions is not a function
[stderr] 
TypeError: config_plugins_1.AndroidConfig.Permissions.withBlockedPermissions is not a function
[stderr] 
    at withImagePicker (/home/expo/workingdir/build/node_modules/expo-image-picker/plugin/build/withImagePicker.js:63:57)
[stderr] 
    at plugin (/home/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/withRunOnce.js:55:25)
[stderr] 
    at withRunOnce (/home/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/withRunOnce.js:41:10)
[stderr] 
    at /home/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/withRunOnce.js:54:12
[stderr] 
    at withStaticPlugin (/home/expo/workingdir/build/node_modules/@expo/prebuild-config/node_modules/@expo/config-plugins/build/plugins/withStaticPlugin.js:157:12)
[stderr] 
    at /home/expo/workingdir/build/node_modules/@expo/prebuild-config/build/plugins/withDefaultPlugins.js:271:50
[stderr] 
    at Array.reduce (<anonymous>)
[stderr] 
    at withOptionalLegacyPlugins (/home/expo/workingdir/build/node_modules/@expo/prebuild-config/build/plugins/withDefaultPlugins.js:265:18)
[stderr] 
    at withLegacyExpoPlugins (/home/expo/workingdir/build/node_modules/@expo/prebuild-config/build/plugins/withDefaultPlugins.js:282:10)
[stderr] 
    at getPrebuildConfig (/home/expo/workingdir/build/node_modules/@expo/prebuild-config/build/getPrebuildConfig.js:76:60)
```

**Comment by jcalem** [comment]
+1 I'm also getting the this issue

**Comment by coroo** [comment]
By default, the config-plugin will be installed using version 3.0.1.

So I solved the issue by install the newest version for expo plugins: `npm i @expo/config-plugins`
or:
```
"dependencies": {
   "@expo/config-plugins": "^6.0.0",
   ...
},
```

### TL;DR;
I manage to check the root cause by checking the `withBlockedPermissions` type and it use the type from the @expo/config-plugins. Turns out, there is no export for that function, only `withPermissions`.
So I check the latest code here https://github.com/expo/expo/blob/main/packages/%40expo/config-plugins/build/android/Permissions.js. It's different with my local `Permission.js`, so I try to install the @expo/config-plugins manually and its work for me.
